spec/mailers/previews/patron_request_mailer_preview.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+# Preview all emails at http://localhost:3000/rails/mailers/patron_request
```

Comment:
When I go to localhost:3000/rails/mailers, the path for patron_request seems to be http://localhost:3000/rails/mailers/patron_request_mailer .  Assume the actions would be filled in with the work on the other issue #2159

Reply:
In the interest of alacrity, I'm going to make a comment on the other confirmation mailer issue to update the URL for this comment.